### PR TITLE
Fixes #16620: Add a command to generate user accounts after installation

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16620

### CODE REVIEW

**Comment by amousset** [comment]
WIP

**Comment by amousset** [comment]
Commit modified

File: share/commands/server-create-user [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    u)
+      USER="${OPTARG}"
+      ;;
+  esac
+done
+
+# check hash
+if ! grep -qE "^[[:space:]]*<authentication[[:space:]]+hash=\"bcrypt\"[[:space:]]*>" "${USERFILE}"
+then
+  echo "This command can only create users with the 'bcrypt' hash method"
```
Comment (peckpeck) [inline share/commands/server-create-user:27]: Maybe a link to some doc or an explanation f what the problem is ?

File: share/commands/server-create-user [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  exit 1
+fi
+
+# check if user is already there
+if grep -qE "name[[:space:]]*=[[:space:]]*\"${USER}\"" "${USERFILE}"
+then
+  echo "User '${USER}' already exists, aborting."
```
Comment (peckpeck) [inline share/commands/server-create-user:34]: cnat' we replace it if it exists ?

Comment (amousset) [reply share/commands/server-create-user:34]: I prefer to avoid messing with xml using sed as much as I can

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**